runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc

```diff
@@ -54,22 +55,31 @@ void ElementwiseActivationLayer::configure(const IPortableTensor *input, IPortab
     case ElementwiseActivationType::kReLU:
       if (input->data_type() == OperandType::FLOAT32)
       {
-        if (alpha == std::numeric_limits<float>::infinity() && beta == 0.f)
+        if ((alpha == std::numeric_limits<float>::infinity() || alpha == 6.0f) && beta == 0.f)
         {
           cpu::ops::ElementwiseActivationLayer::configure(
             input, output, alpha, beta, cpu::ops::ElementwiseActivationType::kReLU);
 
-          _backward_kernel = [](const IPortableTensor *output, const IPortableTensor *incoming,
-                                IPortableTensor *outgoing) {
-            nnfw::cker::train::ReLUGrad(getShape(output), getBuffer<float>(output),
-                                        getShape(incoming), getBuffer<float>(incoming),
-                                        getShape(outgoing), getBuffer<float>(outgoing));
+          auto relu_cker = [&alpha]() {
+            if (alpha == std::numeric_limits<float>::infinity())
+              return nnfw::cker::train::ReLUGrad;
+            else if (alpha == 6.0f)
+              return nnfw::cker::train::ReLU6Grad;
+            else
+              throw std::runtime_error{"no supported relu kernel"};
+          }();
```

Comment:
```suggestion
          }();
```
lambda is invoked here. 
and it returns a function pointer to relu_cker.